content/folderPane.js

```diff
@@ -180,6 +180,6 @@
   };
   
   /* Refresh pane */
-  mainWindow.gFolderTreeView.mode = mainWindow.gFolderTreeView.mode;
```

Comment:
This line was added by the commit '[Fix folder sort on startup](https://github.com/protz/Manually-Sort-Folders/commit/68314ddf4556244e54c5ed13e2b3db5b402c5d0f)'
But this is causing the exception on restarting TB78.
See if this is necessary in TB78 as well. If so, we will need to find out alternative solution.